pro/src/screens/OfferEducationalStock/FormStock/__specs__/FormStock.spec.tsx

```diff
@@ -0,0 +1,87 @@
+import '@testing-library/jest-dom'
+
+import { render, screen, waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Form, Formik } from 'formik'
+import React from 'react'
+
+import {
+  EducationalOfferType,
+  Mode,
+  OfferEducationalStockFormValues,
+} from 'core/OfferEducational'
+import { validationSchema } from 'screens/OfferEducationalStock/validationSchema'
+import { SubmitButton } from 'ui-kit'
+
+import FormStock, { IFormStockProps } from '../FormStock'
+
+const renderFormStock = ({
+  initialValues,
+  onSubmit = jest.fn(),
+  props,
+}: {
+  initialValues: OfferEducationalStockFormValues
+  onSubmit: () => void
+  props: IFormStockProps
+}) => {
+  return render(
+    <Formik
+      initialValues={initialValues}
+      onSubmit={onSubmit}
+      validationSchema={validationSchema}
+    >
+      <Form>
+        <FormStock {...props} />
+        <SubmitButton isLoading={false}>Enregistrer</SubmitButton>
+      </Form>
+    </Formik>
+  )
+}
+
+describe('TimePicker', () => {
+  let initialValues: OfferEducationalStockFormValues
+  const onSubmit = jest.fn()
+  let props: IFormStockProps
+
+  beforeEach(() => {
+    props = {
+      mode: Mode.CREATION,
+    }
+    initialValues = {
+      eventDate: '',
+      eventTime: '',
+      numberOfPlaces: '',
+      totalPrice: '',
+      bookingLimitDatetime: null,
+      priceDetail: '',
+      educationalOfferType: EducationalOfferType.SHOWCASE,
+    }
+  })
+
+  it('should display field error when field empty', async () => {
```

Comment:
C'est plus facile pour un dev qui découvre le code de comprendre quand ce sont de "vraies phrases" `should display an error when the field is empty`